website/license.html

```diff
@@ -1,5 +1,25 @@
 <!DOCTYPE html>
 <html>
+  <style>
+    @media (prefers-color-scheme: dark) {
+      a:link {
+        color: #268bd2;
+      }
+      a:visited {
+        color: #6c71c4;
+      }
+      body {
+        background-color: #002b36;
+        color: white;
+      }
+    }
+    @media (prefers-color-scheme: light) {
+      body {
+        background: #fff;
+        color: #333;
+      }
+    }
```

Comment:
Isn't this already in main.css? Why duplicate the rules?

Reply:
Duplicated rules have been deleted. I left <style></style> tags just in-case you end up changing style later on. However, probably unnecessary.